.github/workflows/connect-dev-release.yml

```diff
@@ -0,0 +1,102 @@
+# TODO: add tests and call it [Build/Test]
+name: "[Build] connect"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  pull_request:
+    paths:
+      - "packages/blockchain-link"
+      - "packages/connect/**"
+      - "packages/connect-common/**"
+      - "packages/connect-iframe/**"
+      - "packages/connect-web/**"
+      - "packages/transport"
+      - "packages/utxo-lib"
+      - "packages/utils"
+      - "submodules/trezor-common"
+      - "yarn.lock"
+
+env:
+  DEV_SERVER_HOSTNAME: "dev.suite.sldev.cz"
+
+jobs:
+  build-connect-explorer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::538326561891:role/gh_actions_trezor_suite_dev_deploy
+          aws-region: eu-central-1
+
+      - name: Extract branch name
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        id: extract_branch
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Build connect-web
+        run: |
+          git submodule update --init --recursive
+          yarn install --immutable
+          yarn build:libs
+          yarn workspace @trezor/connect-web build
+          ./packages/connect-web/scripts/check-inline-build-size.sh
+          yarn workspace @trezor/connect-iframe build
+          yarn workspace @trezor/connect-popup build
+          yarn workspace @trezor/connect-webextension build
+
+      - name: Build connect-explorer
+        env:
+          ASSET_PREFIX: /connect/${{ steps.extract_branch.outputs.branch }}
+          # TODO: maybe we don't want to build it for production for every branch
+          NODE_ENV: "production"
+          name: $CI_COMMIT_REF_NAME
+          url: https://$DEV_SERVER_HOSTNAME/connect/$CI_COMMIT_REF_NAME
+        run: |
+          yarn workspace @trezor/connect-explorer build
+
+      - name: Build connect-examples webextension
+        run: |
+          node ./packages/connect-examples/update-webextensions.js --trezor-connect-src "https://$DEV_SERVER_HOSTNAME/connect/${CI_COMMIT_REF_NAME}/"
```

Comment:
Same here, if you wan to use the path and have it in the /branch-name/ use this `${{ steps.extract_branch.outputs.branch }}`

Reply:
Done. https://github.com/trezor/trezor-suite/pull/11207/commits/aa703cec1d77f2381ab7fc3ad1f49f971c00b9aa